godot-bindings/src/godot_version.rs

```diff
@@ -52,6 +54,21 @@ pub fn parse_godot_version(version_str: &str) -> Result<GodotVersion, Box<dyn Er
     })
 }
 
+pub(crate) fn validate_godot_version(godot_version: &GodotVersion, version_str: &str) {
+    assert_eq!(
+        godot_version.major,
+        4,
+        "Only Godot versions with major version 4 are supported; found version {}.",
+        version_str.trim()
+    );
+
+    assert!(
+        godot_version.minor > 0,
+        "Godot 4.0 is no longer supported by godot-rust; found version {}.",
+        version_str.trim()
+    );
+}
```

Comment:
Is `godot_version.full_string` not enough to output the original string?

If the regex match logic is correct, it should return the original input, trimmed.

Reply:
yeah, that was exceptionally silly: 

![image](https://github.com/user-attachments/assets/1d0b38dc-f8fc-45a3-8b63-ade541e2308e)